src/components/RiskAssessmentForm.js

```diff
@@ -31,22 +31,14 @@ class RiskAssessmentForm extends Component {
 
     onRadioInputChange = (e) => {
         let patientInfo = this.props.patientInfo;
-        patientInfo[e.currentTarget.name] = e.currentTarget.value;
-        this.props.setPatientInfo(patientInfo);
-
-    }
-    onTextInputChange = (e) => {
-        let patientInfo = this.props.patientInfo;
-        patientInfo[e.currentTarget.name] = e.currentTarget.value;
+        patientInfo[e.currentTarget.name] = JSON.parse(e.currentTarget.value); // Using Boolean instead of string
```

Comment:
boolean won't work for gender and rtPCR conducted which has more than 2 values

Reply:
updated it, only doing JSON.parse for true false values